release-notes-v1.1.8.md

```diff
@@ -0,0 +1,59 @@
+v1.1.8
+
+This release contains a reference implementation of [FIDO Device Onboard (FDO) Specification](https://fidoalliance.org/specs/FDO/FIDO-Device-Onboard-PS-v1.1-20220419/).
+
+It includes the below components:
+  * Protocol Reference Implementation (PRI): [pri-fidoiot](https://github.com/fido-device-onboard/pri-fidoiot) is a JAVA based implementation of all the components specified in the FDO Specification. It supports the following cryptographic modes.
+    * Signing keys: ECDSA NIST P-256, ECDSA NIST P-384, RSA2048RESTR, and Intel EPID 1.1.
+    * Key Exchanges: ECDH256, ECDH384, ASYMKEX2048, ASYMKEX3072, DHKEXid14, and DHKEXid15.
+    * Ciphers: AES128/CTR/HMAC-SHA256, AES128/HMAC-SHA256, AES256/CTR/HMAC-SHA384, AES256/HMAC-SHA384, AES-CCM-64-128-128, AES-CCM-64-128-256, AES128GCM, AES256GCM, and RSA/NONE/OAEPWithSHA256AndMGF1Padding.
+    * Public Key Encoding: Crypto, X509, COSEKey , X5 Chain.
+    * COSE Signature Types: ES256, ES384, RS2048.
+
+  * Client SDK: [client-sdk-fidoiot](https://github.com/fido-device-onboard/client-sdk-fidoiot) is a C based implementation for the device component specified in the FDO Specification. Additionally, it supports an implementation of the device that uses the TPM infrastructure. It supports the following cryptographic modes.
+    * Signing keys: ECDSA NIST P-256, and ECDSA NIST P-384
+    * Key Exchanges: ECDH256, and ECDH384
+    * Ciphers: AES-CCM-64-128-128, AES-CCM-64-128-256, AES128GCM, and AES256GCM.
+    * Public Key Encoding:  X509.
+    * COSE Signature Types: ES256, and ES384.
+
+  * EPID Verification Service: [epid-verification-service](https://github.com/fido-device-onboard/epid-verification-service) is a wrapper service written on top of the EPID SDK to assist the FDO Rendezvous service and FDO Owner service to perform device signature verification for EPID based devices.
+  
+### New Features
+
+**client-sdk-fidoiot**: Securing the device credentials (secrets) inside TPM as per the spec.
+
+**client-sdk-fidoiot**: Support for FIDO Service Info Module (FSIM) added.
+
+### Changes to existing features
+
+**pri-fidoiot**: Added REST API support to update the replacement RV information.
+
+### Fixed Issues
+ 
+**client-sdk-fidoiot**: Cross attestation between CSDK client and PRI services are fixed.
+
+**client-sdk-fidoiot**, **pri-fidoiot** **epid-verification-service**: Updated the required third party dependencies to be complaint with FIPS.

```

Comment:
Let FIPS be in changed features and we need to add the disclaimer for it .. Let us add "The version of third-party dependencies have been updated." here